elastic/kibana
# [Table list view] Improve UX (phase 1)

This PR is the **phase 1** for the work to enhance the UX of the `<TableListView />`. It consist mainly in converting the component exposed by the `kibana_react` plugin into a package, refactor and cleanup the interface and integrate the package into

* Dashboard
* Visualisation
* Maps
* Graphs

It contains the following intermediate PR that **have already been reviewed**

* [x] https://github.com/elastic/kibana/pull/135807
* [x] https://github.com/elastic/kibana/pull/136002
* [x] https://github.com/elastic/kibana/pull/138244
* [x] https://github.com/elastic/kibana/pull/139556
* [x] https://github.com/elastic/kibana/pull/140040


## How to test

* Launch the cloud deployment
* Navigate to the list view of the above apps and make sure that everything works as expected, mainly:
  *  search filter
  * tags
  * description
  * navigate to details / edit view
